Resources/Prototypes/_Goobstation/Maps/submarine.yml

```
@@ -0,0 +1,67 @@
+        - type: StationNameSetup
+          mapNameTemplate: '{0} Submarine {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: 'DV'
+        #- type: StationEmergencyShuttle
+          #emergencyShuttlePath: /Maps/Shuttles/DeltaV/NTES_Propeller.yml
+        - type: StationJobs
+          overflowJobs:
+            - Passenger
+          availableJobs:
+          #civilian
```

yamllint (1.35.1)

[warning] missing starting space in comment

(comments)